leap.js

```diff
@@ -23,7 +23,7 @@ function Year(year) {
  * Whether given year is a leap year.
  */
 Year.prototype.isLeap = function () {
-//
+  return ((year % 4 == 0) && (year % 100 !== 0)) || (year % 400 == 0);
```

Comment:
`ReferenceError: year is not defined` would say that you use a variable without create it before !
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/ReferenceError

I advice you to use to read this comment:
```javascript
 // (you have to use this.year)
```
Because I use `this.year` in my test, which exist in the scope of the `Year` function :wink: !